### DISCUSSION TITLE
What can we do to improve security on a general level?

### DISCUSSION THREAD

**Discussion by spoenemann** [eclipse-oct/open-collaboration-tools#72] [General]
The OCT project team recently proposed to move this project to the Eclipse Foundation – [you can read the full proposal here](https://projects.eclipse.org/proposals/eclipse-open-collaboration-tools). One of the reasons for this move is to take advantage of the foundation's experience regarding security aspects. We welcome any kind of support for improving the overall security of OCT.

@mbarbero could you give us a few pointers where we could start the journey to better security for this open source project?

CC @msujew @kineticsquid

**Comment by mbarbero** [comment]
Hi Miro, sorry for the delayed response, this notification got lost in my inbox.

We have some initial material in the [Security Handbook](https://eclipse-csi.github.io/security-handbook/) and in our [specifications of levels for better supply chain security](https://github.com/eclipse-csi/gradually/blob/main/spec.md). Both documents are still a work in progress, so feel free to open issues on their respective repositories or ask more general questions at the [GH organization level](https://github.com/orgs/eclipse-csi/discussions). FYI, there’s already a [discussion thread](https://github.com/orgs/eclipse-csi/discussions/3) about the security levels document. HTH

**Reply by spoenemann** [reply]
Thanks, we'll have a look!
